lib/ember_cli/shell.rb

```diff
@@ -54,7 +55,20 @@ def spawn(command)
     end
 
     def exec(command)
-      Kernel.system(env, command, process_options) || exit(1)
+      options = process_options.dup
+      out = options.delete(:out)
+      Open3.popen2e(env, command, options) do |stdin, stdout_and_stderr, wait_thread|
```

Comment:
Unused block argument - `stdin`. If it's necessary, use `_` or `_stdin` as an argument name to indicate that it won't be used.<br>Line is too long. [85/80]